applications_tests/gls_vans_3d/packed_bed.output

```diff
@@ -0,0 +1,24 @@
+Running on 1 MPI rank(s)...
+   Number of active cells:       640
+   Number of degrees of freedom: 3300
+   Volume of triangulation:      5.656854249e-05
+
+*****************************************************************
+Transient iteration : 1        Time : 0.002    Time step : 0.002    CFL : 0.06743077177
+*****************************************************************
+
+*****************************************************************
+Transient iteration : 2        Time : 0.004    Time step : 0.002    CFL : 0.248173394
+*****************************************************************
+
+*****************************************************************
+Transient iteration : 3        Time : 0.006    Time step : 0.002    CFL : 0.2453174007
+*****************************************************************
+
+*****************************************************************
+Transient iteration : 4        Time : 0.008    Time step : 0.002    CFL : 0.2122950793
+*****************************************************************
+
+*****************************************************************
+Transient iteration : 5        Time : 0.01     Time step : 0.002    CFL : 0.224382112
+*****************************************************************
```

Comment:
We should write some simulation information at the end of the test. For example, you can see the poiseuille3d application test, for which cells error_velocity and error_pressure are written in the output file.